appengine-deploy/README.md

```diff
@@ -0,0 +1,137 @@
+<!--
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# appengine-deploy
+
+This action deploys your source code to App Engine and makes the URL available to
+later build steps via outputs. This allows you to parameterize your App Engine
+deployments.
+
+**Note** This action will install gcloud in the background if not using in with
+the [`setup-gcloud` action](../setup-gcloud/README.md).
+
+## Prerequisites
+
+This action requires Google Cloud credentials that are authorized to deploy an
+App Engine Application. See the Authorization section below for more information.
```

Comment:
Consider adding a link to App Engine documentation here.

Reply:
done.